File: framework/languages/java/samples/java/TicTacToe/src/main/java/systems/zlink/samples/tictactoe/server/configuration/SampleTopology.java
package systems.zlink.samples.tictactoe.server.configuration;

public final class SampleTopology {
    public static final String ApiEndpoint = "inproc://zlink-java-sample-tictactoe-api";
    public static final String PlayStreamEndpoint = "inproc://zlink-java-sample-tictactoe-stream";

    private SampleTopology() {
    }
}

File: framework/languages/java/samples/kotlin/TicTacToe/src/main/kotlin/systems/zlink/samples/kotlin/tictactoe/server/configuration/SampleTopology.kt
package systems.zlink.samples.kotlin.tictactoe.server.configuration

object SampleTopology {
    const val ApiEndpoint: String = "inproc://zlink-kotlin-sample-tictactoe-api"
    const val PlayStreamEndpoint: String = "inproc://zlink-kotlin-sample-tictactoe-stream"
}
